### PR TITLE
Add manual install page for Kubernetes docs.

### PR DESCRIPTION
**Pull request by evilnick**
## Done
 Added a new page to Kubernetes docs
 Also added to navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/kubernetes/docs/install-manual](http://0.0.0.0:8001/kubernetes/docs/install-manual)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)

### CODE REVIEW

File: templates/kubernetes/docs/install-manual.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+applications:
+  aws-integrator:
+    charm: cs:~containers/aws-integrator
+    num_units: 1
+relations:
+  - ['aws-integrator', 'kubernetes-master']
+  - ['aws-integrator', 'kubernetes-worker']
+  ```
+
+You can also [download the fragment here][asset-aws-overlay]
```
Comment (anthonydillon) [inline templates/kubernetes/docs/install-manual.md]: The full stop is missing at the end of this sentence.

File: templates/kubernetes/docs/install-manual.md [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+details on the format used by Juju, see the [Juju bundle documentation][juju-bundle].
+
+#### Retrieving a bundle from a running model
+
+Sometimes a more convenient way of getting a local bundle file which matches
+exactly the deployment you want is simply to save a running model as a bundle.
+This will preserve configuration, relations and the charms used in the
+deployment so a structural replica can be recreated.
+
+This can be done dimply by running the command:
```
Comment (anthonydillon) [inline templates/kubernetes/docs/install-manual.md]: s/dimply/simply

File: templates/kubernetes/docs/install-manual.md [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+
+Running this command will output some login information and a URL for the GUI
+interface (the GUI actually runs on the Juju controller instance). On visiting
+the URL given and logging in, a graphical representation of the current model
+will be shown. To export the model as a YAML bundle, click on the `Export`
+button near the top left of the screen.
+
+![][image-gui]
+
+For more information on the Juju GUI, see the [Juju documentation][juju-gui]
```
Comment (anthonydillon) [inline templates/kubernetes/docs/install-manual.md]: Full stop missing.

File: templates/kubernetes/docs/install-manual.md (hunk 1 above)
Comment (evilnick) [reply templates/kubernetes/docs/install-manual.md]: :+1:

File: templates/kubernetes/docs/install-manual.md (hunk 2 above)
Comment (evilnick) [reply templates/kubernetes/docs/install-manual.md]: :+1:

**Comment by evilnick** [comment]
@anthonydillon Thanks! I remembered to add it to the sitemap and everything this time

**Comment by anthonydillon** [comment]
@evilnick thanks LGTM. Merge away.

**Pull request merged**